src/js/components/App.js

```diff
@@ -41,7 +41,10 @@ export default React.createClass({
           </div>
           <div className="col-md-9">
             <ArticleForm show={this.state.edit} current={this.state.current} />
-            {this.state.current ? <Viewer {...this.state.current} /> : null}
+            {!this.state.current ? null :
```

Comment:
the ``!`` is visually hard to catch :) Putting null at the end was so problematic ?